client/core/core.go

```diff
@@ -4619,11 +4702,15 @@ func (c *Core) PreOrder(form *TradeForm) (*OrderEstimate, error) {
 // Trade is used to place a market or limit order.
 func (c *Core) Trade(pw []byte, form *TradeForm) (*Order, error) {
 	// Check the user password.
-	crypter, err := c.encryptionKey(pw)
-	if err != nil {
-		return nil, fmt.Errorf("Trade password error: %w", err)
+	var crypter encrypt.Crypter
+	if len(pw) > 0 {
```

Comment:
Will you please comment on this and the related change in prepareTrackedTrade?

Also, the diff in prepareTrackedTrade would be been a lot tighter if `prepareWallet` simply returned early if `crypter == nil`.